content/direction/enablement/delivery/index.md

```diff
@@ -8,37 +8,35 @@ This page outlines the vision, strategy, and goals of the [Delivery team](../../
 
 ## Mission
 
-Enable any Sourcegraph customer or user to trial or run (in production) Sourcegraph in a way fits within their environment, supports their level of technical expertise and allows them to easily access the value that our product provides. Make it as easy as possible to access this value, and to maintain it going forwards by easily staying up to date with Sourcegraph's latest developments.
+Enable any Sourcegraph prospect or customer to use Sourcegraph in a way fits within their environment and allows them to easily access the value that our product provides. Make it as easy as possible for customers to access the value of Sourcegraph's latest developments through a simple upgrade process
```

Comment:
@dan-mckean - tried to leave the message in tact and just reduced some of the words.